server/game/drawcard.js

```diff
@@ -223,6 +225,10 @@ class DrawCard extends BaseCard {
         return Math.max(0, baseStrength + this.dominanceStrengthModifier);
     }
 
+    getIcons() {
+        return _.filter(Icons, icon => this.hasIcon(icon));
```

Comment:
What other icons are in the icons collection that necessitates this filter?

Reply:
Not sure I understand this one...

Reply:
Similar issue to above, weird looking code because of how icons is implemented